apps/site/src/pages/[shortname]/types/entity-type/[...slug-maybe-version].page/entity-type-form.tsx

```diff
@@ -0,0 +1,249 @@
+import {
```

Comment:
I know you are aware there are visual oddities that require fixing and I don't suggest we tackle them in this PR but just flagging for posterity. Some font sizes in this menu are quite different from the original designs.

![image](https://user-images.githubusercontent.com/37453800/220110297-3e2d4053-f474-463e-af79-ba0f9c12da33.png)

![image](https://user-images.githubusercontent.com/37453800/220109765-cfb21300-6d97-484c-93d1-4fb37e692f68.png)